examples/gno.land/r/demo/banktest/z_1_filetest.gno

```diff
@@ -7,10 +7,9 @@ import (
 )
 
 func main() {
-	banktestAddr := std.DerivePkgAddr("gno.land/r/banktest")
+	banktestAddr := std.DerivePkgAddr("gno.land/r/demo/banktest")
 
 	// simulate a Deposit call.
-	std.TestSetOrigPkgAddr(banktestAddr)
```

Comment:
We should not remove this testing case.

Reply:
0dd861efce7035e74befec96b395624539730134